README.md

```diff
@@ -66,20 +68,23 @@ The following configuration options are available-
 <thead>
   <tr>
     <th>Key</th>
-    <th>Explaination</th>
+    <th>
+    <!-- Adding &nbsp to increase column width. -->
+    Explaination&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
```

Comment:
This makes the Explanation be aligned left while the rest is aligned right.
If you want to increase the column with you should use proper html for that, not nbsp characters.

Reply:
> If you want to increase the colum

Updated to use css.
Earlier I tried to do it with CSS but was testing it on vscode's markdown preview where it wasn't working, but it's working on Github so that's fine.